cypress/support/fragments/exportManager/exportManagerSearchPane.js

```diff
@@ -105,6 +105,13 @@ export default {
       searchButton.click(),
     ]);
   },
+  searchByAuthorityControl() {
+    waitClick();
+    cy.do(jobTypeAccordion.find(Checkbox({ id: 'clickable-filter-type-auth-headings-updates' })).click());
+  },
+  downloadLastCreatedJob(jobId) {
+    cy.get(`a:contains(${jobId})`).first().click()
```

Comment:
leave TODO: here to rework this code with interactors in the future

Reply:
done